index.js

```diff
@@ -100,6 +100,7 @@ function send(ctx, path, opts) {
     ctx.set('Cache-Control', 'max-age=' + (maxage / 1000 | 0));
     ctx.type = type(path);
     ctx.body = fs.createReadStream(path);
+    ctx.length = stats.size;
```

Comment:
Actually, you can still get `ctx.length` after `ctx.set('Content-Length', stats.size)`,

`ctx.length = n` just a alias to `set('Content-Length', n)`